data_model/src/isi.rs

```diff
@@ -207,26 +231,22 @@ mod transparent {
     }
 
     macro_rules! impl_into_box {
-        (
-            $($isi:ident $(< $($generic:ident $(< $nested_generic:ident >)?),+ >)?)|*
-            ==> $boxed:ident :: $variant:ident
+        ( $($isi:ty)|*
+          => $middle:ty => $boxed:ty[$variant:ident],
+          => $middle_ref:ty => $boxed_ref:ty[$variant_ref:ident]
```

Comment:
```suggestion
          => $middle_ref:ty => $boxed_ref:ty :: $variant_ref:ident
```

Is that not allowed?

Reply:
well, type cannot be followed by `::` because it's a path delimiter